src/components/orders/orders.js

```diff
@@ -27,19 +27,16 @@ function getTotalPrice() {
 export function incrementCounter(event) {
     if (event.target.classList.contains('order-card__btn-plus')) {
         const dataFromLocalStorage = getOrdersDataToStorage('Orders');
-        const eventBtn = event.target;
-        const orderCardId = +event.currentTarget.id;
-        const elemCounter = eventBtn.parentNode.parentNode;
-        const valueCount = elemCounter.childNodes[1].children[0];
-        const valueTotalPriceDish = elemCounter.parentNode.parentNode.children[2].children[1].children[0];
+        const orderCard = event.currentTarget;
+        const orderCardId = +orderCard.id;
+        const valueCount = orderCard.children[2].children[0].childNodes[1].children[0];
 
         dataFromLocalStorage.forEach(item => {
             if (item.id === orderCardId) {
                 item.count += 1;
                 valueCount.textContent = item.count;
                 setOrdersDataToStorage(dataFromLocalStorage);
-                const resultPriceDish = +valueTotalPriceDish.textContent + item.price;
-                valueTotalPriceDish.textContent = resultPriceDish;
+                getTotalPriceForOneDish(event, item);
```

Comment:
set orderCard, not event!!
the same valueTotalPriceDish (you should just set valueTotalPriceDish, that's all)